Wiki Reorg: Nominator Page (#4418)
w3f/polkadot-wiki: docs/learn/learn-nominator.md

:::

If you landed on this page, you decided to understand how you can be a good nominator. Note, this
page is not for [nomination pool](./learn-nomination-pools.md) members, although pool members might
gain essential knowledge about how to choose nomination pools.

The information provided on this page is complementary to that on the
[**Staking Page**](./learn-staking.md) and [**Advanced Staking Page**](./learn-staking-advanced.md).
Make sure you read those pages as well before nominating.

> ## Why Nominating?

Reviewer: ```suggestion
## Why Nominate?
```

> - You become part of the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} Movement, a

Reviewer: ```suggestion
- You become part of the {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} movement, a
```

>   group of professionals and enthusiasts around the world aspiring to build and foster the next-gen

Reviewer: ```suggestion
  group of diverse professionals and enthusiasts around the world aspiring to build and foster the next-gen
```

>   Internet, Web 3.0: a decentralized, privacy-focused, and trustless internet.

Reviewer: ```suggestion
  Internet, Web3: a decentralized, privacy-focused, and trustless internet.
```

- You are an essential piece of the puzzle, keeping
  {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} secure, deciding the future
  direction of the project by voting in [OpenGov](./learn-opengov.md) and participating in parachain

>   [crowdloans](./learn-crowdloans.md).

Reviewer: This statement can be rephrased as it has less to do with nominating DOT/KSM. You can't participate in crowdloans and take part in governance. What you could highlight is here is that "As a nominator securing the network, you can also use the nominated DOT/KSM to participate in Governance"

```suggestion
- You are an essential piece of the puzzle, keeping
  {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} secure, deciding the future
  direction of the project by voting in [OpenGov](./learn-opengov.md) and participating in parachain
  [crowdloans](./learn-crowdloans.md).
```

> - You start to understand how {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} works

Reviewer: ```suggestion
- You will start to understand how {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} works
```

  at a technical-level. When you feel comfortable with your nomination skills and knowledge, you can
  open your [nomination pool](./learn-nomination-pools.md), help others secure the network and earn

>   rewards, and build your reputation as a trusted nominator. If you need more than this, the next

Reviewer: ```suggestion
  rewards, and build your reputation as a trusted nomination pool operator. If you like to be more involved, the next
```

  step is to become a [validator](./learn-validator.md).
- By getting [staking](./learn-staking.md) rewards you fight
  {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} inflation and (likely) your

>   currency inflation as well.

Reviewer: ```suggestion
- By getting [staking](./learn-staking.md) rewards you keep up with or (likely) stay ahead of 
  {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} inflation.
```

are 'locked' (bonded). You can
[stop nominating at any time](../maintain/maintain-guides-how-to-nominate-polkadot.md), but remember
that the action is effective in the next era and does not automatically unbond your funds. Unbonding
is a separate action, and it takes effect after the unbonding period, which is
{{ polkadot: 28-day long on Polkadot :polkadot }}{{ kusama: 7-day long on Kusama :kusama }}. This is
calculated by taking the **bonding duration** (in eras), multiplying it by the **length of a single
era** (in hours), and dividing by the **hours in a day** (24). Example:
({{ polkadot: 28 × 24 ÷ 24 = 28 days :polkadot }}{{ kusama: 28 × 6 ÷ 24 = 7 days :kusama }}). A
staking lock will be visible on the Polkadot-JS UI during the unbonding period, and after it, the
staking lock can be unlocked, and the bonded funds become free balance you can transfer.

## Setting-up Accounts

### Stash, Controller & Staking Proxy

The first thing you need to do before becoming a nominator is to make sure you have a
[**stash account**](./learn-staking.md/#stash-and-controller-accounts-for-staking) where you can

> transfer funds you want to use for staking. For these accounts, it is recommended a "cold solution"

Reviewer: ```suggestion
transfer funds you want to use for staking. For these accounts, it is recommended to use a "cold wallet" solution 
```

such as [Ledger](../general/ledger.md) or
[Parity Signer](./learn-account-generation.md#parity-signer).

After setting up the stash account, it is recommended to have a
[**controller account**](./learn-staking.md/#stash-and-controller-accounts-for-staking) or a
[**staking proxy**](./learn-staking-advanced.md/#staking-proxies) (or both, controller and staking
proxy). Although you can be a nominator with just a stash account, having at least a controller or a
staking proxy is good practice for security reasons.

:::info In the near future, Controller Accounts will be deprecated

The concept of a controller account is very similar to that of a staking proxy: isolate the stash

> account but use its economic power and sign staking-related transactions on behalf of it. This is

Reviewer: ```suggestion
account but sign staking-related transactions on behalf of it. This is
```

### Rewards Payout Account

As a nominator, you will be asked to choose an account where rewards will be paid. You can select
one of the following options:

- back to staking: rewards are compounded to the bonded amount.
- to stash: rewards are sent to the stash account as a free balance.
- to the controller: rewards are sent to the controller account as a free balance.
- to another account: rewards are sent to a user-defined account (not stash or controller).

:::info

Being a nominator is made simpler by using the
[**Staking Dashboard**](https://staking.polkadot.network/#/overview) that will guide you step by
step through creating a stash-controller relationship, specifying rewards destination, and
nominating validators (more on this below). Note that staking proxies are not currently supported on
the dashboard.

:::

## Nominating with the Polkadot-JS UI

### Targets Page

There are many factors to consider when deciding which of your nominations. One helpful tool to
choose validators is the Staking [Targets](https://polkadot.js.org/apps/#/staking/targets) table in
the Polkadot-JS UI. This allows sorting validators using various metrics. Below are the relevant

> metrics, followed by a brief description of each.

Reviewer: ```suggestion
metrics shown as an example, followed by a brief description of each.
```

  validators must be able to run at break-even to continue operations sustainably. Independent
  validators that rely on the commission to cover their server costs help to keep the network
  decentralized. Some validators, operated by central exchanges, etc., keep 100% of the commission
  to payout their staking service clients and therefore do not provide any rewards to external
  nominators. The commission is just one piece of the puzzle you should consider when picking
  nominating validators.
- **total stake**: The total amount of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :Kusama }} tokens
  staked by nominators and the validator (i.e. own stake, see below).
- **own stake**: The amount of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :Kusama }} tokens the
  validator has put up as a stake. A higher own stake can be considered as having more "skin in the
  game". This can imply increased trustworthiness. However, a validator not having a large amount of
  "own stake" is not automatically untrustworthy, as the validator could nominate from a different
  address.
- **return**: Average annual yield paid out to nominators (i.e. number of rewards divided by the

>   number of bonded tokens). Note that nominating those with a higher yield does not guarantee

Reviewer: ```suggestion
  number of bonded tokens). Note that nominating those with a higher yield may not guarantee
```

  on the Support Pages.
- The
  ["Why am I not receiving staking rewards?"](https://www.reddit.com/r/Polkadot/comments/10kurje/why_am_i_not_receiving_staking_rewards/)
  Reddit article.
- The ["Why am I not receiving staking rewards?"](./learn-staking.md#why-am-i-not-receiving-rewards)
  section on the Staking Page.

:::

Rewards are _lazy_ - somebody must trigger a payout for a validator for rewards to go to all of the
validator's nominators. Any account can do this, although validator operators often do this as a
service to their nominators. See the page on [Simple Payouts](learn-staking-advanced.md) for more
information and instructions for claiming rewards.

:::note Explainer videos on Nominating

These concepts have been further explained in the following videos:

- [Why Nominate on Polkadot & Kusama](https://www.youtube.com/watch?v=weG_uzdSs1E&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8&index=4)
- [What to Consider when Nominating Validators on Polkadot and Kusama](https://www.youtube.com/watch?v=K-a4CgVchvU&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8&index=9)
- [Nominating/Staking on Polkadot and Kusama](https://youtu.be/FCXC0CDhyS4)

:::

> ## Good Practices

Reviewer: ```suggestion
## Good Nominator Practices
```

nominators in this era, for instance, is unlikely to have more than
{{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={512}/> :polkadot }}
{{ kusama: <RPC network="kusama" path="consts.staking.maxNominatorRewardedPerValidator" defaultValue={512}/> :kusama }}
in the next era. However, an already-oversubscribed validator with 1000 nominators this era is very
likely to be oversubscribed in the next era as well.

If you are not nominating with a large number of DOTs, you should try to avoid
[oversubscribed](../general/glossary.md#oversubscribed) validators. It is not always easy to
calculate if the validator selected will be oversubscribed in the next session; one way to avoid
choosing potentially oversubscribed validators is to filter out any that are
[at capacity](../general/glossary.md#capacity) on the Targets page.

Finally, if you have a minimal amount of DOTs, you may need more time to fit your nomination into
the election set. The nominator-to-validator mapping has to fit in a single block, and if there are
too many nominators, the lowest-staked nominations will be dropped. This value is dynamic and will

> vary over time. You can read the blog post

Reviewer: ```suggestion
Finally, if you have a minimal amount of DOTs close to the value of `minActiveNomination`, you may need to stake more DOT to get into
the election set. The nominator-to-validator mapping solution needs to be evaluated within a single block duration, and if there are
too many nominators, the lowest-staked nominations will be dropped from even being considered to be part of the electing set. This `minActiveNomination` value is dynamic and will
vary over time. You can read the blog post
```
